test/snippet/range/views/deep_with_param.cpp

```diff
@@ -1,8 +1,8 @@
 #include <vector>
 
 #include <seqan3/alphabet/nucleotide/dna5.hpp>
+#include <seqan3/io/detail/take_view.hpp>
 #include <seqan3/range/views/deep.hpp>
```

Comment:
```suggestion
#include <seqan3/alphabet/nucleotide/dna5.hpp>
#include <seqan3/range/views/deep.hpp>
```

Reply:
I don't know, who or what includes "take_view.hpp", but seems it is used on purpose in this snippet, shouldn't it also be explicit included?

Reply:
We don't need it because we use `std::views::take`

Reply:
If you rebase, the snippet changes

Reply:
hehe, yes I just notice :-) :+1: 
*eseiler is rebasing and compiling 277'000 lines in his head faster than my 8 core machine*